design-tokens/tokens/layer1/base.json

```diff
@@ -1,109 +1,121 @@
-        "type": "spacing"
+      "150": {
+        "value": "1.25rem",
+        "type": "dimension"
       },
-      "md": {
-        "value": "{size.dimension.md}",
-        "type": "spacing"
+      "200": {
+        "value": "1rem",
+        "type": "dimension"
       },
-      "lg": {
-        "value": "{size.dimension.lg}",
-        "type": "spacing"
+      "300": {
+        "value": "1.5rem",
+        "type": "dimension"
       },
-      "xl": {
-        "value": "{size.dimension.xl}",
-        "type": "spacing"
+      "400": {
+        "value": "2rem",
+        "type": "dimension"
       },
-      "multi-value": {
-        "value": "{size.dimension.sm} {size.dimension.xl}",
-        "type": "spacing",
-        "description": "You can have multiple values in a single spacing token. Read more on these: https://docs.tokens.studio/available-tokens/spacing-tokens#multi-value-spacing-tokens"
-      }
-    },
-    "borderRadius": {
-      "sm": {
-        "value": "4",
-        "type": "borderRadius"
+      "500": {
+        "value": "2.5rem",
+        "type": "dimension"
       },
-      "lg": {
-        "value": "8",
-        "type": "borderRadius"
+      "600": {
+        "value": "3rem",
+        "type": "dimension"
       },
-      "xl": {
-        "value": "16",
-        "type": "borderRadius"
+      "800": {
+        "value": "4rem",
+        "type": "dimension"
+      }
+    },
+    "sizes": {
+      "global": {
+        "xs": {
+          "value": "0.25rem",
+          "type": "dimension"
+        }
       },
-      "multi-value": {
-        "value": "{size.borderRadius.sm} {size.borderRadius.lg}",
-        "type": "borderRadius",
-        "description": "You can have multiple values in a single radius token. Read more on these: https://docs.tokens.studio/available-tokens/border-radius-tokens#single--multiple-values"
+      "icons": {
+        "sm": {
+          "value": "1rem",
+          "type": "dimension"
+        },
+        "md": {
+          "value": "1.25rem",
+          "type": "dimension"
+        },
+        "lg": {
+          "value": "1.5rem",
+          "type": "dimension"
+        }
       }
     }
   },
-  "opacity": {
-    "low": {
-      "value": "10%",
-      "type": "opacity"
+  "borderRadius": {
+    "sm": {
+      "value": "2",
+      "type": "borderRadius"
+    },
+    "lg": {
+      "value": "8",
+      "type": "borderRadius"
     },
     "md": {
-      "value": "50%",
-      "type": "opacity"
+      "value": "4",
+      "type": "borderRadius"
+    },
+    "none": {
+      "value": "0",
+      "type": "borderRadius"
     },
-    "high": {
-      "value": "90%",
-      "type": "opacity"
+    "circle": {
+      "value": "700",
+      "type": "borderRadius"
     }
   },
   "fontFamilies": {
-    "heading": {
-      "value": "Inter",
+    "mono": {
+      "value": "Ubuntu Mono",
       "type": "fontFamilies"
     },
-    "body": {
-      "value": "Roboto",
+    "sans": {
+      "value": "Inter",
```

Comment:
We need fallback font families, I suggest we use the same as tailwind for that:
https://tailwindcss.com/docs/font-family

Reply:
Thanks. Ill add those Tailwind values there

Reply:
In our last discussion didn't you say you couldn't do it in figma?

Reply:
Yes. This token cannot be used in Figma. Figma has a fallback font built-in. But I can add it as a string so you guys can use it in code.

Reply:
Ok but that's gonna be problematic if you can't use it in figma right?
I think we are better off adding it only on our side directly.

Reply:
No, it's ok. Don't worry.